lib/tasks/schools_data.rake

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+namespace :schools_data do
+  HEADERS = %w[urn name town postcode lead_school open_date close_date].freeze
+  REFINED_CSV_PATH = Rails.root.join("data/schools.csv").freeze
+
+  # This task requires two csvs, see readme under "Regenerating data/schools.csv" for details
+  desc "Build school csv with only the required columns from the establishment and lead school csv"
+  task :build_csv, %i[establishment_csv_path lead_schools_csv_path output_path] => [:environment] do |_, args|
```

Comment:
`build` vs `generate`

Inclined to call it `generate` since there is meaning associated to this task.